src/uct/ib/dc/verbs/dc_verbs.c

```diff
@@ -284,6 +301,8 @@ ucs_status_t uct_dc_verbs_ep_put_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov, siz
                        "uct_dc_verbs_ep_put_zcopy");
     UCT_CHECK_LENGTH(uct_iov_total_length(iov, iovcnt), UCT_IB_MAX_MESSAGE_SIZE,
                      "put_zcopy");
+    UCT_CHECK_LENGTH(uct_iov_total_lazy_length(iov, iovcnt),
```

Comment:
need to unite those 3 macros (check iov size, length, and lazy length) to single macro, instead of calling each of them in every function